### DISCUSSION TITLE
Variable in notice

### DISCUSSION THREAD

**Discussion by loan-talvat** [directus/directus#14819] [Feature Requests]
It would be perfect to be able to put variables in a notice 

![image](https://user-images.githubusercontent.com/23056644/182590753-d6e4bc0b-fdd4-413b-9767-b2df1e743038.png)

**Comment by rijkvanzanten** [comment]
So that `{{id}}` would just be the value of the `id` field in the current form?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
